src/common/graphcmn.cpp

```diff
@@ -631,8 +631,8 @@ void wxGraphicsContext::GetDPI( wxDouble* dpiX, wxDouble* dpiY)
     {
         // Use some standard DPI value, it doesn't make much sense for the
         // contexts not using any pixels anyhow.
-        *dpiX = 72.0;
-        *dpiY = 72.0;
+        *dpiX = 96.0;
+        *dpiY = 96.0;
```

Comment:
Are you sure about this? I think it's used for Cairo too and it definitely uses 72 and not 96.

Reply:
Yes, I definitely want 96 here on Windows. This is also what [D2D ](https://docs.microsoft.com/en-us/windows/win32/api/d2d1/nf-d2d1-id2d1rendertarget-getdpi) and [GDI+](https://docs.microsoft.com/en-us/windows/win32/api/gdiplusgraphics/nf-gdiplusgraphics-graphics-getdpix) return. I'll investigate Cairo.